README.md

```diff
@@ -163,5 +154,66 @@ start();
   </tr>
 </table>
 
+## Contributing
+
+You want this package to be awesome and we want to deliver on that. As you know
+already you can just [File A Ticket](#file-a-ticket), but thats not actually the
+best way for you to get what you need (read on to see why). The best way is for
+you to [Be A Champion](#be-a-champion) and [dive into the code](#where-do-you-start).
+
+#### File A Ticket
+
+The reality is that filing a ticket isn't always enough. **This is probably only
+going to work if your issue aligns with both the interests _and_ the resources available** to the core team. Here are the things that align with our _interests_
+in order of priority.
+
+1. Fixing **_widespread, common, and critical Bugs_**.
+2. Fixing **_uncommon but necessary Bugs_**.
+3. Introducing new  **_Features that have broad value_**.
+
+Now keep in mind available resources. Long story short, the thing you care about
+needs to be cared about by either a lot of other people, or by us.
+
+**BUT!** There's another and, arguably even **_better way_** that helps you get what
+you need faster: [Be A Champion](#be-a-champion).
+
+#### Be A Champion
+
+Being a _champion_ makes it easy for us to help you. Which is what we all want!
+So how can you be a champion? [Sponsor the Project](https://github.com/sponsors/DoubleSymmetry) or _be willing to write some code_.
+
+**If _you're willing_** to write some code **_we're willing_** to:
+
+- Open a design discussion, give feedback, and approve something that works.
+- Provide guidance in the implementation journey.
+
+So, in a nutshell, let us know you're willing to do the work and ask for a little
+guidance, and watch the things you care about get done faster then anyone else.
+The best help will be given to those who are willing to help themselves.
+
+###### You don't have experience you say? It's OK!
+
+You may be thinking that you can't help because you know nothing about native
+iOS or Android or maybe even React code. But we're willing to help teach you.
```

Comment:
```suggestion
iOS or Android or maybe even React code. But we're willing to help guide you.
```
Don't want to overpromise that we can teach everything about a platform to someone, but we'll all do our best to help :)